docs/fsharp/tutorials/type-providers/creating-a-type-provider.md

```diff
@@ -14,12 +14,17 @@ ms.assetid: 82bec076-19d4-470c-979f-6c3a14b7c70a
 
 # Tutorial: Creating a Type Provider
 
-> [!NOTE]
-This guide was written for F# 3.0 and will be updated.
+The type provider mechanism in F# is a significant part of its support for information rich programming. This tutorial explains how to create your own type providers by walking you through the development of several simple type providers to illustrate the basic concepts. For more information about the type provider mechanism in F#, see [Type Providers](index.md).
 
-The type provider mechanism in F# 3.0 is a significant part of its support for information rich programming. This tutorial explains how to create your own type providers by walking you through the development of several simple type providers to illustrate the basic concepts. For more information about the type provider mechanism in F#, see [Type Providers](index.md).
+The F# ecosystem contains a range of type providers for commonly used Internet and enterprise data services. For example:
 
-F# 3.0 contains several built-in type providers for commonly used Internet and enterprise data services. These type providers give simple and regular access to SQL relational databases and network-based OData and WSDL services. These providers also support the use of F# LINQ queries against these data sources.
+- [FSharp.Data](http://fsharp.github.io/FSharp.Data/) includes type providers for JSON, XML, CSV and HTML document formats
```

Comment:
nit: we're using https links whenever they're accepted. I just fixed all of them in the F# directory yesterday.